scifight/admin.py

```diff
@@ -6,6 +8,27 @@
 admin.AdminSite.site_header = 'SciFight'
 
 
+class TournamentSpecificModelAdmin(admin.ModelAdmin):
+
+    def get_queryset(self, request):
+        self.scifight_user = request.user
+        self.exclude_tournament_field()
+        qs = super().get_queryset(request)
+        if not request.user.is_superuser:
+            qs = qs.filter(tournament=request.user.userprofile.tournament)
+        return qs
+
+    def exclude_tournament_field(self):
```

Comment:
This method should be private, so prefix it's name with an underscore. Or simply put these three lines into `get_queryset`.